assistant-app.js

```diff
@@ -1639,12 +1639,12 @@ class AssistantApp {
   /**
    * Constructs LineItem with chainable property setters.
    *
-   * @param {string} name Name of the line item.
    * @param {string} id Unique identifier for the item.
+   * @param {string} name Name of the line item.
    * @return {LineItem} Constructed LineItem.
    */
-  buildLineItem (name, id) {
-    return new LineItem(name, id);
+  buildLineItem (id, name) {
```

Comment:
This also seems to be a breaking change. Could be a revision for the next major version release

Reply:
Here's the constructor for `LineItem`:
```
onstructor (lineItemId, name) {